api/components/image.schema.json

```diff
@@ -12,16 +12,71 @@
             ]
         },
         "path": {
-            "description": "The image path from the resources folder",
+            "description": "The image path from the resources folder or from the network depending on the value of fromNetwork.",
             "type": "string"
         },
+        "fromNetwork": {
+            "description": "Whether the image will be loaded from a network url of from an application's resource path.",
+            "type": "boolean",
+            "default": false
+        },
         "width": {
-            "description": "The image width",
+            "description": "The image width.",
             "type": "number"
         },
         "height": {
-            "description": "The image height",
+            "description": "The image height.",
             "type": "number"
+        },
+        "alignment": {
+            "description": "How to align the image.",
+            "$ref": "defs/alignment.schema.json"
+        },
+        "centerSlice": {
+            "description": "The center slice for a nine-patch image.",
+            "$ref": "defs/rect.schema.json"
+        },
+        "errorBuilder": {
+            "description": "The error callback when the image encounters an error during loading.",
+            "type": "component"
+        },
+        "excludeFromSemantics": {
+            "description": "Whether to exclude this image from semantics.",
+            "type": "boolean",
+            "default": false
+        },
+        "filterQuality": {
+            "description": "The quality of the image.",
+            "$ref": "defs/filterQuality.schema.json"
+        },
+        "fit": {
+            "description": "How the image should fit the parent box.",
+            "$ref": "defs/boxFit.schema.json"
+        },
+        "frameBuilder": {
+            "description": "A callback that handles the creation of the widget that represents the image.",
```

Comment:
Like above, this is not a callback but a component.
Also i'm not sure when this is used. 
Is it used when the image is loading ? Or is it a component that wrap the image inside ?
If so, how do the dev specify where the image should be in the frame ?